traffic_ops/traffic_ops_golang/ats/atsserver/ipallowdotconfig.go

```diff
@@ -77,26 +80,45 @@ func GetIPAllowDotConfig(w http.ResponseWriter, r *http.Request) {
 // GetChildServers returns the child servers of the given Mid serverName. This should not be called with an Edge server.
 func GetChildServers(tx *sql.Tx, serverName tc.CacheName) (map[tc.CacheName]atscfg.IPAllowServer, error) {
 	qry := `
-SELECT
-  s.host_name,
-  s.ip_address,
-  COALESCE(s.ip6_address, '')
-FROM
-  server s
-  JOIN type tp on tp.id = s.type
-  JOIN cachegroup cg on cg.id = s.cachegroup
-WHERE
-  (tp.name = '` + tc.MonitorTypeName + `' OR ( tp.name LIKE '` + tc.EdgeTypePrefix + `%')
-  AND cg.id IN (
-    SELECT
-      cg2.id
-    FROM
-     server s2
-     JOIN cachegroup cg2 ON (cg2.parent_cachegroup_id = s2.cachegroup OR cg2.secondary_parent_cachegroup_id = s2.cachegroup)
-    WHERE
-      s2.host_name = $1 )
-  )
-`
+	SELECT
```

Comment:
Rather than fixing up deprecated endpoints we should consider just removing these now, since we already know we need to get rid of them. That seems more practical than keeping these alive just to keep the ATS config functionality via TP going and keeping CIAB working (which you seem to have fixed already in your ORT.py PR which seems done and just needs review?).

Reply:
ugh, yeah, you're right. The thing is that the ORT.py PR is waiting on my giving-ORT-its-own-build PR which in turn is waiting on the apache org to set up some dockerhub repos, which - finally - is waiting on me to give them a list of DockerHub users to add as admins for the repos they will create. Which I was just reminded of seconds ago, as it happens, and am about to do.

Reply:
If we removed the routes now, how badly would CIAB fail? Obviously the caches wouldn't be able to handle requests, but is there anything else? Does it prevent other service from starting?

If is just breaks the caching in CIAB but all the other services start up fine, I don't really see a problem breaking that functionality temporarily (as well as the TP functionality). Most of the automated testing I know about that depends on CIAB doesn't require the caches to successfully handle requests (unless you count the readiness container).

Reply:
It is pretty much just the caches. The only thing that's truly concerning, then, is the readiness container - which established that CiaB is "ready" if and only if DS URL curls succeed. I guess it's not a huge deal just to break caching - although that does give me a nice, warm feeling sometimes when I'm testing things with far-reaching changes - but I know that I have tools already that rely on readiness, and I just hope no one else does.

Reply:
Alright, I tore the affected endpoints out.